polygeist/lib/Dialect/Polygeist/Transforms/LoopInternalization.cpp

```diff
@@ -63,24 +63,17 @@ bool isCandidateFunction(FunctionOpInterface func) {
 /// A loop nest is a candidate iff is perfect and contains only affine or scf
 /// for loops.
 bool isCandidateLoopNest(LoopLikeOpInterface loop) {
-  if (!LoopTools::isOutermostLoop(loop)) {
-    LLVM_DEBUG(llvm::dbgs() << "not candidate: not outermost loop\n");
+  if (!LoopTools::isOutermostLoop(loop))
```

Comment:
remove debug messages, as they are now misleading after we changed to only print candidate loops.